cobigen-eclipse/cobigen-eclipse-test/.classpath

```diff
@@ -5,6 +5,7 @@
 	<classpathentry kind="src" path="src/main/java/"/>
 	<classpathentry exported="true" kind="lib" path="lib/slf4j-api-1.7.21.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/assertj-core-3.8.0.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/core-test-5.1.0.jar"/>
```

Comment:
Do you know why this jar has been added to your classpath?

Reply:
I don't know. I saw i had an error in cobigen-eclipse-test in Jenkins and in my local eclipse there was this classpath additional and i tried to add this also and see if this makes some difference. Normaly I changed there nothing, therfore i didn't commit it first.